mmdet/ops/nms/src/nms_cpu.cpp

```diff
@@ -66,6 +66,149 @@ at::Tensor nms(const at::Tensor& dets, const float threshold) {
   return result;
 }
 
+template <typename scalar_t>
```

Comment:
We can add a line "Modified from https://xxxx, Soft-NMS is added." at the top of this file.